Story/StoryHooks.cs

```diff
@@ -809,5 +819,16 @@ private bool PlayersStandingStill(On.RegionGate.orig_PlayersStandingStill orig,
             }
             return orig(self);
         }
+
+        private void GhostHunch_Update(On.GhostHunch.orig_Update orig, GhostHunch self, bool eu)
+        {
+            orig(self, eu);
+            if (OnlineManager.lobby == null) return;
+            if (!OnlineManager.lobby.isOwner && OnlineManager.lobby.gameMode is StoryGameMode)
+            {
+                if (self.ghostNumber != null && (self.room.game.session as StoryGameSession).saveState.deathPersistentSaveData.ghostsTalkedTo[self.ghostNumber] > 0)
```

Comment:
Just curious: Is this method ran every frame in the game and this is trying to optimize so it only RPCs when a change occurs or is there another reason it's set to > 0 to prevent early RPC in the interaction?

Reply:
yup it runs every frame so this just waits until the echo has been activated before sending the rpc